agent/bin/smoke_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Please add docs for this script.

Reply:
Is this expected to run manually during the provisioning of the scripts or as part of the health checks?

Reply:
Added, manual run.